ejb3/src/main/java/org/jboss/as/ejb3/component/stateful/cache/StatefulSessionBeanCache.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2022, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.jboss.as.ejb3.component.stateful.cache;
+
+import java.util.function.Supplier;
+
+import org.wildfly.clustering.ee.Restartable;
+import org.wildfly.clustering.ejb.bean.BeanStatistics;
+import org.wildfly.clustering.ejb.remote.AffinitySupport;
+
+/**
+ * A stateful session bean cache.
+ * Any {@link StatefulSessionBean} retrieved from this cache *must* invoke either {@link StatefulSessionBeanInstance#close()}, {@link StatefulSessionBeanInstance#remove()}, or {@link StatefulSessionBeanInstance#discard()} when complete.
```

Comment:
did you mean StatefulSessionBean#close(), StatefulSessionBean#remove(), StatefulSessionBean#discard()?

Reply:
Yes.  I'll fix that.

Reply:
Fixed.